<ui version="4.0" >
 <class>pqPointSpriteDisplayPanelDecorator</class>
 <widget class="QGroupBox" name="pqPointSpriteDisplayPanelDecorator" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PointSprite</string>
  </property>
  <property name="title" >
   <string>PointSprite</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="ModeLabel" >
     <property name="text" >
      <string>Point Sprite Mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QComboBox" name="RenderMode" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="TextureLabel" >
     <property name="text" >
      <string>Point Sprite Texture</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="pqPointSpriteTextureComboBox" name="TextureCombo" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="MaxPixelLabel" >
     <property name="text" >
      <string>Max pixel size</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QSpinBox" name="MaxPixelSize" >
     <property name="toolTip" >
      <string>Set the maximum size of the sprites in pixels. Hardware limitations may further reduce this value.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="ScaleArrayLabel" >
     <property name="text" >
      <string>Scale by</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="pqDisplayArrayWidget" native="1" name="ScaleBy" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Select method for coloring dataset geometry.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QStackedWidget" name="RadiusStack" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="MappingRadiusPage" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>0</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="1" >
        <widget class="QPushButton" name="RadiusMapping" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>Edit Radius Transfer Function</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ConstantRadiusPage" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="RadiusLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string>Radius</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="pqDoubleEdit" name="RadiusEdit" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="OpacityArrayLabel" >
     <property name="text" >
      <string>Opacity by</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2" >
    <widget class="pqDisplayArrayWidget" native="1" name="OpacityBy" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Select method for coloring dataset geometry.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3" >
    <widget class="QStackedWidget" name="OpacityStack" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="MappingOpacityPage" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>0</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QPushButton" name="OpacityMapping" >
         <property name="text" >
          <string>Edit Opacity Transfer Function</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ConstantOpacityPage" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="OpacityLabel" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string>Opacity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="OpacitySpin" >
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqDisplayArrayWidget</class>
   <extends>QWidget</extends>
   <header>pqDisplayArrayWidget.h</header>
  </customwidget>
  <customwidget>
   <class>pqDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>pqDoubleEdit.h</header>
  </customwidget>
  <customwidget>
   <class>pqPointSpriteTextureComboBox</class>
   <extends>QComboBox</extends>
   <header>pqPointSpriteTextureComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
